api/test/routes/homepage.test.js

```diff
@@ -0,0 +1,79 @@
+/**
+ * Tests the endpoint "/homepage".
+ */
+const supertest = require('supertest');
+
+const Team = require('../../src/models/team.model');
+const {
+  connectDb,
+  closeConnection,
+  clearAllCollections,
+  deleteDb,
+  seedDb,
+} = require('../util/db');
+const app = require('../../src/server');
+
+const request = supertest(app);
+
+/// The database used for this test suite.
+const DB_NAME = 'researchify_test_homepage';
+/// The homepage route prefix.
+const ROUTE_PREFIX = '/homepage';
+
+beforeAll(async () => {
+  await connectDb(DB_NAME);
+});
+
+afterAll(async () => {
+  await deleteDb();
+  await closeConnection();
+});
+
+beforeEach(async () => {
+  await seedDb();
+});
+
+afterEach(async () => {
+  await clearAllCollections();
+});
+
+describe('POST /homepage/:team_id', () => {
+  it('should return 404 for a non-existent team', async () => {
+    const res = await request.post(`${ROUTE_PREFIX}/613b888ffca059539f01fc64`);
```

Comment:
Dumb question: Why are we specifying this particular object id? Will MongoDB never generate this id when we're creating the database for this endpoint?

Reply:
Good Q.  I'm counting on this to be unique enough that we never generate this ID again. We need to meet the hexadecimal and length requirements so just chose an arbitrary id that's long enough.